tutorials/Python/setup.md

```diff
@@ -0,0 +1,157 @@
+# Setup instructions for the Oxford Computational Biochemistry Python course
+
+Included here are instructions to help you get started with the Oxford
+Computational Biochemistry Python course. The instructions are split into
+two parts:
+
+1. Setup
+
+These instructions should be followed prior to the tutorial. These will walk
+you through creating the appropriate conda environment for the tutorial and
+testing that everything works as expected.
+
+2. Starting the tutorial
+
+These instructions should be followed on the day of / when starting the tutorial.
+We assume here that you have completed the setup instructions on the same machine
+and that testing passed as expected.
+
+
+## 1. Setup
+
+Here we assume you have completed the main setup instructions and have done the
```

Comment:
```suggestion
Here we assume you have completed the main setup instructions (see `setup.md`) and have done the
```

Reply:
I won't commit the suggestion here just because I think we can make that a link too.